recipes-bsp/acpi-tables/files/edison/acpi-tables-load

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# This script copies all ACPI tables into /kernel/firmware/acpi
```

Comment:
Should it say "from /kernel/firmware/acpi" instead?

Reply:
Certainly. I've fixed it now in my local branch. It will appear when I force push next version.